lib/write-type.js

```diff
@@ -52,29 +52,28 @@ function getWriteType(options) {
   }
 
   function number(encoder, value) {
-    var ivalue = value | 0;
```

Comment:
This should be leaved as is: it has the same semantics, but there are performance consideration to take into account. See https://jsperf.com/math-floor-alternatives